core/modules/pty/__init__.py

```diff
@@ -9,8 +9,16 @@
 
 class PTYController:
     def __init__(self, configs):
-        self.start_agent()
+        self.supported_architectures = {
```

Comment:
Şurada aarch32 olması gerekiyor mu?

Reply:
32bit Ubuntu ile Raspbian OS (32bit) dağıtımları üzerinde platform.machine() çıktısı aynı oluyor (RPi4 için -> armv6l).